jvb/src/main/java/org/jitsi/videobridge/websocket/ColibriWebSocket.java

```diff
@@ -103,5 +109,6 @@ public interface EventHandler {
          * @param ws the {@link ColibriWebSocket} from which a message was received.
          */
         void webSocketTextReceived(ColibriWebSocket ws, String message);
+        default void webSocketError(Throwable cause) {}
```

Comment:
Should have javadoc.  I'd also say it should take the ws as an argument, to match the other methods.